doc/rbd/nvmeof-initiator-linux.rst

```diff
@@ -0,0 +1,80 @@
+==============================
+ NVMe/TCP Initiator for Linux
+==============================
+
+Prerequisites
+=============
+
+-  Red Hat Enterprise Linux 9.2 or later
```

Comment:
This is upstream documentation, so it would be nice to avoid the impression that RHEL is required.

@oritwas What is the specific patchset/feature that requires RHEL 9.2 as opposed to e.g. any RHEL 9 or CentOS Stream 9?  What is the corresponding requirement for mainline kernels?

Reply:
Add the upstream kernel version and the other distros.
As for RHEL the NVMe/TCP initiator was GA in RHEL 9.0 but got broken in RHEL 9.1 :(
We highly recommend using the one from RHEL 9.2